docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -192,9 +199,9 @@ For example, to find the *Data Generator* source connector, type *data* in the s
 +
 Click the card to select the connector, and then click *Next*.
 
-. For the *Kafka instance*, click the card for the {product-kafka} instance that you configured for Connectors, and then click *Next*.
+. For the *Kafka instance*, click the card for the {product-kafka} instance that you configured for connectors, and then click *Next*.
```

Comment:
Per MAS style discussions, don't make a proper name (for example, a parameter, variable, or field name) do double duty as a regular noun in a sentence. This causes problems for translation.

In this case, it's easily solved by removing "the" before `*Kafka instance*`.